files/tasks.py

```
@@ -1351,6 +1362,31 @@ def subscribe_user(email, name):
+        if response.status_code == 200:
+            logger.info(f"Newsletter subscription successful for {email} (Country: {country})")
+            return True
+        else:
+            logger.warning(
+                f"Newsletter subscription failed for {email}. "
+                f"Status: {response.status_code}, Response: {response.text[:200]}"
+            )
+            return False
+            
+    except requests.exceptions.Timeout:
+        logger.error(f"Newsletter subscription timeout for {email}")
```

Ruff (0.14.4)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)